content/en/blog/2022-09-01-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,213 @@
+---
+author: developer-guy
+date: 2022-09-01 11:30:00+00:00
+title: Managing Kyverno Policies as OCI Artifacts with OCIRepository Sources
+description: "We will give you a real-world example of the newest feature in Flux v0.32 called OCIRepository sources to show you how you can leverage this feature to manage Kyverno policies as OCI Artifacts."
+url: /blog/2022/08/manage-kyverno-policies-as-ocirepositories
+resources:
+- src: "**.{png}"
+  title: "Image #:counter"
+---
+
+{{< imgproc twitter-ss Resize 400x >}}
```

Comment:
You could use https://gohugo.io/content-management/shortcodes/#tweet as well.

Reply:
thank you so much for your help